build-system/compile/build.conf.js

```diff
@@ -67,6 +67,23 @@ const validTransformedRemovableImports = {
   '../build/ampdoc.css': ['cssText', 'ampDocCss'],
 };
 
+const experimentFlag = (function () {
+  const constant = argv['define_experiment_constant'];
+  if (Array.isArray(constant)) {
+    if (constant.length > 0) {
+      throw new Error('Only one define_experiment_constant flag is allowed');
+    } else {
+      return constant[0];
+    }
+  }
+  return constant;
+})();
+
+const experimentRemovedImports =
+  experimentFlag === 'MOVE_FIXED_LAYER'
+    ? {'./../fixed-layer': ['FixedLayer']}
+    : {'../../../src/service/fixed-layer': ['FixedLayer']};
+
```

Comment:
/cc @jridgewell in case you think this should live elsewhere. (I know, we must refactor this 🙁)

Reply:
the plan is to remove this file completely correct?

Reply:
The plan is to refactor this code into `babel.config.js`

Reply:
It's fine for now, let's just delete this file soon.

Reply:
Refactor has been merged. These changes will need to be moved to [`build-system/babel-config/`](https://github.com/ampproject/amphtml/tree/master/build-system/babel-config).

Reply:
@kevinkimball @jridgewell Refactor was temporarily reverted, but will be merged with fixes shortly. In order to avoid conflicts, we'll either need to hold this back until that's done, or incorporate these changes into the refactor. 

My preference is to do the latter, but let me know if this PR needs to be merged in a hurry.